tests/base/src/test/java/org/wildfly/security/ssl/SSLAuthenticationTest.java

```diff
@@ -17,12 +17,7 @@
  */
 package org.wildfly.security.ssl;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.fail;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
```

Comment:
Hello @nidhiazad , this PR looks great! 

Just a total minor. I can still see this asterisk here, this will make the build fail as we are using a checkstyle plugin to avoid  the asterisk imports. Please use the individual classes names when importing and update this PR

Let us know if you need any help with that ! Thank you!

Reply:
Fixing this

Reply:
I have made the changes. Kindly review and let me know if anything needs to be done.

Reply:
Hi @nidhiazad, thanks for the updates! In general, we try to squash all the commits for a given issue into a single one. Could you do that for this PR? 

Let me know if you need any assistance in doing so!